### ISSUE TITLE
Font fingerprinting breaks the `font-face` property

### ISSUE THREAD

**Issue by savchenko** [sereneblue/chameleon#443]
## Expected Behavior

Enabling "spoof font fingerprint" does not break basic font definitions (sans, serif, monospace, etc)

## Current Behavior

From [webbrowsertools.com/font-fingerprint](https://webbrowsertools.com/font-fingerprint):
```
Basic font list Support Metrics Metrics
>>                              
default         ✕       N/A     N/A
cursive         ✕       N/A     N/A
fantasy         ✕       N/A     N/A
monospace       ✕       N/A     N/A
sans-serif      ✕       N/A     N/A
serif           ✕       N/A     N/A
```

Example screenshot:
![image](https://user-images.githubusercontent.com/300146/90311335-6dda7100-dee9-11ea-8a26-849a7fcdb265.png)

## Relevant settings

[chameleon_1597487289000.json.log](https://github.com/sereneblue/chameleon/files/5078482/chameleon_1597487289000.json.log)

This can be reproduced with `browser.display.use_document_fonts` set to either `1` or `0`.

## Context (Environment)

FF 79.0
Chameleon v0.20.24

Might be related to https://github.com/sereneblue/chameleon/issues/360

**Comment by sereneblue** [comment]
@asvc I'm able to replicate your example screenshot of Github by changing the about:config preference you mentioned above, but not with Chameleon. 

I did encounter the same issue with https://webbrowsertools.com/font-fingerprint. It does seem Chameleon is missing some font faces but I'm a bit confused about the lack of serif support. That should be supported. I'm looking into this.

**Comment by asvc** [comment]
With Chameleon v0.21.0, `browser.display.use_document_fonts` is set to `1` (default).

"Spoof font fingerprint" enabled:
![image](https://user-images.githubusercontent.com/300146/90637373-6f72a480-e21b-11ea-8f63-a7ef3f6b673f.png)

**Comment by Z0pyrus** [comment]
I have the same issue, but not on all sites. On those where I encountered this, i added them to a whitelist were Enable spoof client rects is turned off

**Comment by sereneblue** [comment]
@savchenko This is now resolved in v0.22.22.
